### ISSUE TITLE
M16 - Icone de notificação estático

### ISSUE THREAD

**Issue by juniopereirab** [mia-ajuda/Documentation#156]
### Descrição:
O ícone de notificações permanece inalterado se não houver novas notificações que não foram visualizadas

**Comment by AnaCarolcs** [comment]
@juniopereirab is this enhancement done?

**Comment by juniopereirab** [comment]
> @juniopereirab is this enhancement done?

No, I believe that we have discussed about this, and we have a doubt about the app performance, which make this issue to not been assigned to anyone.

**Comment by AnaCarolcs** [comment]
This issue was prioritized in R2, but we removed it.

**Comment by devsalula** [comment]
Make the description better.
And this is still not make sense for R2. It will be considered in future versions

**Comment by devsalula** [comment]
@juniopereirab Could you make the description a little bit better to understand, please?

**Comment by juniopereirab** [comment]
> @juniopereirab Could you make the description a little bit better to understand, please?

Done!
